packages/shared/styles/components/organisms/SfHeader.scss

```diff
@@ -85,6 +85,7 @@
   &__search {
     margin: var(--header-search-margin, var(--spacer-xs) 0);
     height: 2.25rem;
+    width: var(--header-search-width, 100%);
```

Comment:
but, you should use it here flex because sf-header use `display:flex` `flex: 0 0 100%` for mobile and `flex: 0 1 20rem` for desktop :) wdyt?

Reply:
@justyna-13

Reply:
@pspaczek you're right :)